Makefile

```diff
@@ -361,6 +361,24 @@ test_coverage:
 	fi; \
 	py.test --create-db --cov=awx --cov-report=xml --junitxml=./reports/junit.xml $(TEST_DIRS)
 
+## Run coverage check (compares current to devel)
+check_coverage:
+	@if [ "$(VENV_BASE)" ]; then \
+		. $(VENV_BASE)/awx/bin/activate; \
+	fi; \
+	if [ "${GIT_BRANCH}" == "devel" ] ; then \
+		echo "Doesn't make much sense to compare devel to devel"; \
+		exit 0; \
+	fi;
+	rm -f reports/coverage.xml
+	#-py.test --create-db --cov=awx --cov-report=xml --junitxml=./reports/junit.xml $(TEST_DIRS)
+	#mv reports/coverage.xml reports/current.xml
+	git checkout devel
```

Comment:
What if, instead of `devel`, you used `$(git merge-base --fork-point devel)`

Because people commonly push to branches which are slightly out-of-date with devel, and if you use _current_ devel, you will show a decrease because unrelated PRs have merged since the active PR was opened, and force a rebase to get it right. I don't want to add more arbitrary reasons to need a rebase.

Reply:
Good call on this, let me give it a try.